src/AppBundle/Service/UserRegistrationService.php

```diff
@@ -45,7 +45,7 @@ public function selfRegisterUser(SelfRegisterData $selfRegisterData)
         $existingClient = $this->em->getRepository('AppBundle\Entity\Client')->findOneByCaseNumber(CasRec::normaliseCaseNumber($selfRegisterData->getCaseNumber()));
 
         // ward off non-fee-paying codeps trying to self-register
-        if ($isMultiDeputyCase && $existingClient instanceof Client) {
+        if ($isMultiDeputyCase && ($existingClient instanceof Client) && $existingClient->hasDeputies()) {
```

Comment:
Force of habit, I always put each condition in their own set of brackets in case anyone adds an or condition and a bug is introduced. Not necessary with all &&'s